.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
- [ ] Your linters are fine but as I can see your `.github/workflows/linters.yml` is present only in the `development` branch, since this is the branch that you are working on, please add the above mentioned linter in this branch